modules/tinymce/src/themes/silver/test/ts/browser/sidebar/SidebarShowOptionTest.ts

```diff
@@ -39,6 +39,8 @@ describe('browser.tinymce.themes.silver.sidebar.SidebarShowOptionTest', () => {
   };
 
   const assertNotAnimating = () => UiFinder.notExists(SugarBody.body(), '.tox-sidebar--sliding-growing');
+  const assertSlotContainerOpen = () => UiFinder.exists(SugarBody.body(), '.tox-sidebar--sliding-open');
+  const assertSlotContainerNotOpen = () => UiFinder.notExists(SugarBody.body(), '.tox-sidebar--sliding-open');
```

Comment:
This isn't asserting the slot contain is open, it's checking the sidebar so the should probably be called `assertSidebarOpen` and `assertSidebarClosed` (or similar).